Wikipedia/Code/WMFExploreCollectionViewController.m

```diff
@@ -314,6 +324,13 @@ - (nullable NSURL *)imageURLForIndexPath:(NSIndexPath *)indexPath {
         }
         articleURL = [[content[indexPath.row] featuredArticlePreview] articleURL] ?: [[[content[indexPath.row] articlePreviews] firstObject] articleURL];
         width = self.traitCollection.wmf_nearbyThumbnailWidth;
+    } else if ([section contentType] == WMFContentTypeOnThisDayEvent) {
```

Comment:
This whole block can be removed since there's no lead image on the card, returning nil is expected for this case.